OrbitCore/LinuxUprobesUnwindingVisitorTests.cpp

```diff
@@ -5,6 +5,119 @@
 
 #include "LinuxUprobesUnwindingVisitor.h"
 
+namespace {
+Timer MakeTestTimer(pid_t tid, uint64_t begin_timestamp, uint64_t end_timestamp,
+                    uint64_t function_address) {
+  Timer timer;
+  timer.m_TID = tid;
+  timer.m_Start = begin_timestamp;
+  timer.m_End = end_timestamp;
+  timer.m_FunctionAddress = function_address;
+  return timer;
+}
+}  // namespace
+
+TEST(UprobesTimerManager, OneUprobe) {
+  constexpr pid_t tid = 42;
+  bool process_uretprobes_res;
+  Timer processed_timer;
+  UprobesTimerManager timer_manager;
+
+  timer_manager.ProcessUprobes(tid, 1, 100);
+
+  process_uretprobes_res =
+      timer_manager.ProcessUretprobes(tid, 2, &processed_timer);
+  EXPECT_TRUE(process_uretprobes_res);
```

Comment:
ASSERT_TRUE (if call failed there is no point in checking the rest)

Reply:
Changed it.